src/Frame.js

```diff
@@ -0,0 +1,71 @@
+function Frame() {
+  this.goCount = 0;
+  this.startingPins = 10;
+  this.standingPins = 10;
+  this.rolls = [];
+}
+
+Frame.prototype.registerGo = function(rollScore) {
```

Comment:
It would be nice to have functions named a little closer to the game domain itself.